plugins/com.google.cloud.tools.eclipse.appengine.validation.test/src/com/google/cloud/tools/eclipse/appengine/validation/PomXmlValidatorTest.java

```diff
@@ -58,15 +58,15 @@
   public static void setUp() throws CoreException {
     project = projectCreator.getProject();
     createFolders(project, new Path("src/main/webapp/WEB-INF"));
-    resource = project.getFile("src/main/webapp/WEB-INF/web.xml");
-    ((IFile) resource).create(new ByteArrayInputStream(new byte[0]), true, null);
+    webXmlFile = project.getFile("src/main/webapp/WEB-INF/web.xml");
+    ((IFile) webXmlFile).create(new ByteArrayInputStream(new byte[0]), true, null);
   }
   
   
   @After
   public void tearDown() throws CoreException {
-    if (resource != null) {
-      resource.deleteMarkers(IMarker.PROBLEM, true, IResource.DEPTH_ZERO);
+    if (webXmlFile != null) {
```

Comment:
rather than cleaning up the file at the end of the test, can we make this a non-static instance variable? Static test data risks polluting one test with the next.